components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/java/org/wso2/carbon/inbound/ui/internal/InboundManagementClient.java

```diff
@@ -406,4 +407,35 @@ public String getKAFKASpecialParameters() {
         }
         return specialParamsList;
     }
+
+    public String getKAFKATopicListParameters() {
+        String topicListParams = "";
+        loadProperties();
+        if (prop != null) {
+            String strKey = "kafka.mandatory";
+            String strLength = prop.getProperty(strKey);
+            Integer iLength = null;
+            if (strLength != null) {
+                try {
+                    iLength = Integer.parseInt(strLength);
+                } catch (Exception e) {
```

Comment:
Why have we caught generic exceptions? can be used specific NumberFormatException